apps/nsq_to_nsq/nsq_to_nsq.go

```diff
@@ -255,32 +254,28 @@ func hasArg(s string) bool {
 	return argExist
 }
 
-func main() {
-	var selectedMode int
-
-	cCfg := nsq.NewConfig()
-	pCfg := nsq.NewConfig()
+func commandLineValidation() {
 
-	flag.Var(&nsq.ConfigFlag{cCfg}, "consumer-opt", "option to passthrough to nsq.Consumer (may be given multiple times, see http://godoc.org/github.com/nsqio/go-nsq#Config)")
-	flag.Var(&nsq.ConfigFlag{pCfg}, "producer-opt", "option to passthrough to nsq.Producer (may be given multiple times, see http://godoc.org/github.com/nsqio/go-nsq#Config)")
-
-	flag.Parse()
-
-	if *showVersion {
-		fmt.Printf("nsq_to_nsq v%s\n", version.Binary)
-		return
+	if len(topics) == 0 || *channel == "" {
+		log.Fatal("--topic and --channel are required")
 	}
 
-	if *topic == "" || *channel == "" {
-		log.Fatal("--topic and --channel are required")
+	if len(topics) == 1 {
+		if *destTopic == "" {
+			*destTopic = topics[0]
+		}
 	}
 
-	if *destTopic == "" {
-		*destTopic = *topic
+	if len(topics) > 1 {
+		if *destTopic == "" {
+			log.Fatal("--destination-topic must be specified when multi topic is enabled")
```

Comment:
@ploxiln After thinking on the requirement a bit, I feel I need a bit clarification on the desired behavior.

Do we want to relay the topic only to nsq instances that is currently listening to that topic specifically?

For example consider the following

   Producer A (topic1) -> nsq_to_nsq (topic1) -> Consumer A (topic4)
   Producer A (topic1) -> nsq_to_nsq (topic1)-> Consumer B (topic1)

In our example above, Producer A produce topic 1, and nsq_to_nsq listen for topic1, however since Consumer A only listen to topic4, does Consumer A ever get relayed topic1 messages?

Right now nsq_to_nsq forces Consumer A to create a new topic (ie. topic 1) , is that desirable?